spec/migrations/20181001131632_add_conversion_host_id_to_miq_request_tasks_spec.rb

```diff
@@ -0,0 +1,47 @@
+require_migration
+
+describe AddConversionHostIdToMiqRequestTasks do
+  let(:task_stub) { migration_stub(:MiqRequestTask) }
+  let(:host_stub) { migration_stub(:Host) }
+  let(:conversion_host_stub) { migration_stub(:ConversionHost) }
+
+  migration_context :up do
+    it "creates conversion host" do
+      task = task_stub.create!
+      host = host_stub.create!
+      conversion_host = conversion_host_stub.create!(
```

Comment:
You can just do `.create!(:resource => host)` no need to split up the id/type

Reply:
Good question @carbonin. Probably, because I'm dumb :smile:
Removed and spec updated.

Reply:
@agrare It fails if I use `:resource => host`. The error message is:

```ruby
ActiveModel::UnknownAttributeError:
       unknown attribute 'resource' for AddConversionHostIdToMiqRequestTasks::ConversionHost.
```

Reply:
You probably need to add the relation to the stubbed model